src/StatusQ/Components/StatusListItem.qml

```diff
@@ -34,6 +34,9 @@ Rectangle {
 
     radius: 8
 
+    property string itemId: ""
+    property string titleId: ""
```

Comment:
What's this one about? Can you provide some context on why this is needed? Just by looking at it without context, it doesn't make a lot of sense but I'm sure you have your reasons